src/settings/groups/GroupDetailsModel.ts

```diff
@@ -164,6 +186,19 @@ export class GroupDetailsModel {
 		return this.entityClient.update(newGroupInfo)
 	}
 
+	async changeGroupSenderName(newName: string): Promise<void> {
+		if (this.senderName.isLoaded() && this.senderName.getLoaded() === newName) return
+		const mailGroupId = await this.groupInfo.group
+		await locator.mailAddressFacade.setSenderName(mailGroupId, this.getGroupMailAddress(), newName)
+
+		// we may not be a member of the group and therefore won't necessarily receive updates
+		// for updated mailbox properties.
+		this.senderName.reset()
+		// noinspection ES6MissingAwait
+		this.senderName.getAsync()
```

Comment:
but if we *are* a member it will be still cached in the entity rest cache at this point, right?

Reply:
I couldn't make it break but I am not sure why